build.gradle

```diff
@@ -177,7 +177,7 @@ pitest {
   threads = 10
   outputFormats = ['XML', 'HTML']
   timestampedReports = false
-  mutationThreshold = 80
+  mutationThreshold = 88
```

Comment:
This is neither a change request or a suggestion just a note to open a discussion:
Any particular reason for the explicit increase, this property sets the threshold which I believe the programme mandates it is above 80%.

If we set it to 88% while it might be beneficial and good for the team I feel that there might be cases where we might have to write tests just to satisfy a metric. This is probably something we should agree with the team and @raghera perhaps on our W.O.W sessions

Reply:
keep it 80%